qa/L1_videoreader_test/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+pushd ../..
+
+cd docs/examples/video
+
+# Download video sample
+wget -q -O prepared.mp4 https://download.blender.org/durian/trailer/sintel_trailer-720p.mp4
```

Comment:
Maybe we can put it into our test data instead of downloading every time?

Reply:
I though that we might change the video file in a near future, and add more video files when the VideoReader will support multiple container files